src/test/java/org/fundacionjala/coding/jimmy/encryptionTest/CryptoTest.java

```diff
@@ -0,0 +1,144 @@
+package org.fundacionjala.coding.jimmy.encryptiontest;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "CryptoTest.java" should be located in "org/fundacionjala/coding/jimmy/encryptiontest" directory, not in "/home/travis/build/AT-06/coding/src/test/java/org/fundacionjala/coding/jimmy/encryptionTest". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)